src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -143,9 +143,17 @@ def new_wallet(wallet_type):
         action = request.form["action"]
         if action == "importwallet":
             try:
-                wallet_data = json.loads(
-                    request.form["wallet_data"].replace("\\'", "").replace("'", "h")
-                )
+                wallet_data = json.loads(request.form["wallet_data"])
+
+                # Only need to un-escape the descriptor, if there is one (Specter format
+                #   has it but Electrum backup does not.
+                if "descriptor" in wallet_data:
+                    wallet_data["descriptor"] = (
+                        wallet_data.get("descriptor")
```

Comment:
what's the purpose of `get()` here? With the check above it should be safe to use `["descriptor"]`

Reply:
It's mostly habit so that I don't ever try to retrieve something in an unsafe way that might not be there. But totally unnecessary here--and would fail anyway if the get() ever returned `None`.

Updating now.